recipes/pysumreg/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pysumreg" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysumreg-{{ version }}.tar.gz
+  sha256: 4939bec8feddcbeca06bb922062398605b00b5345da98b4eb22d1d77447ce0ed
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools-scm
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.8
+    - pip
+    - setuptools-scm
```

Comment:
These two should not be needed at runtime.
```suggestion
```

Reply:
Ok, I'll change that.